src/Main.hs

```diff
@@ -26,7 +27,8 @@ data Command = Load String
 
 command str = let res = words str
               in case res of
-                   [":load",f] -> Load f
+                   [":load",f] -> Load f (Nothing)
+                   [":load",f, sourceDir] -> Load f (Just sourceDir)
```

Comment:
So, we can use imports from only one directory. I think, that we can create an application which uses, for example, both lists and linear algebra, which are in different folders. We should support this case.